src/components/groups/group.service.ts

```diff
@@ -1,13 +1,32 @@
 import { Request, Response, NextFunction} from 'express'
-
 import { Group } from './group'
 import { User } from '../users/user'
+import markdown from 'markdown-it'
+import sanitizeHtml from 'sanitize-html'
 
+function formatMdToSafeHTML(mdBody: string): string {
+  const dirtyHtml = new markdown().render(mdBody)
+  const cleanHtml = sanitizeHtml(dirtyHtml, {
+    transformTags: {
+      'h1': 'h3',
+      'h2': 'h4',
+      'h3': 'h5',
+      'h4': 'h5'
+    }
+  })
+  return cleanHtml
+}
```

Comment:
This code block is duplicated, you can extract it to a separate file and put it under `src/util`.